lib/generator.js

```diff
@@ -1,37 +1,176 @@
 'use strict';
 
+const common = require('metarhia-common');
+
+const primitives = [
+  'boolean',
+  'null',
+  'undefined',
+  'number',
+  'string',
+  'symbol',
+];
+
+const globalObjects = [
+  'Object',
+  'Array',
+  'Date',
+  'Function',
+  'Map',
+  'WeakMap',
+  'Set',
+  'WeakSet',
+  'Int8Array',
+  'Uint8Array',
+  'Uint8ClampedArray',
+  'Float32Array',
```

Comment:
This list seems to be also missing multiple other common global objects, such as `RegExp`, `ArrayBuffer`, `DataView`, `Int16Array`, `Uint16Array`, `Int32Array`, `Uint32Array`, `Float32Array`, `Float64Array`.